apps/opik-frontend/src/components/pages-shared/automations/AddEditRuleDialog/AddEditRuleDialog.tsx

```diff
@@ -320,38 +320,44 @@ const AddEditRuleDialog: React.FC<AddEditRuleDialogProps> = ({
     };
     const explainer = EXPLAINERS_MAP[expainerIdMap[scope]];
 
+    // Only show "Go to project" button if exactly one project is selected
+    const actions =
+      formProjectIds.length === 1
+        ? [
+            <ToastAction
+              variant="link"
+              size="sm"
+              className="px-0"
+              altText="Go to project"
+              key="Go to project"
+              onClick={() => {
+                navigate({
+                  to: "/$workspaceName/projects/$projectId/traces",
+                  params: {
+                    projectId: formProjectIds[0],
+                    workspaceName,
+                  },
+                  search: {
+                    type: {
+                      [EVALUATORS_RULE_SCOPE.trace]: "traces",
+                      [EVALUATORS_RULE_SCOPE.thread]: "threads",
+                      [EVALUATORS_RULE_SCOPE.span]: "spans",
+                    }[scope],
+                  },
+                });
+              }}
+            >
+              Go to project
+            </ToastAction>,
+          ]
+        : undefined;
```

Comment:
a nit pick: `null` seems a better option to tell React to rener nothing rather than `undefined`

Reply:
<img width="788" height="491" alt="image" src="https://github.com/user-attachments/assets/e357f27f-a043-4bed-9ce9-0876051691fb" />

True, but it's for the toast 😅 

@aadereiko